sdk/workloads/Azure.ResourceManager.Workloads/tests/Tests/PhpWorkloadsCRUDTests.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Workloads.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Workloads.Tests.Tests
+{
+    public class PhpWorkloadsCRUDTests : WorkloadsManagementTestBase
+    {
+        private const string TestResourceNamePrefix = "sdk-tests-dotNet-php-";
+        private readonly string BasePath = System.AppDomain.CurrentDomain.BaseDirectory;
+
+        public PhpWorkloadsCRUDTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [OneTimeTearDown]
+        public void Cleanup()
+        {
+            CleanupResourceGroups();
+        }
+
+        [RecordedTest]
+        [Test]
+        public async Task TestPhpWorkloadsCRUDOperations()
+        {
+            try
+            {
+                Resources.SubscriptionResource subscription = await Client.GetDefaultSubscriptionAsync();
+                Resources.ResourceGroupResource rg = await CreateResourceGroup(
+                    subscription,
+                    TestResourceNamePrefix,
+                    Core.AzureLocation.EastUS);
+
+                var rgName = rg.Data.Name;
+
+                var resourceJson = File.ReadAllText(
+                    Path.Combine(BasePath, @"TestData\PhpWorkloads\PutPhpWorkload.json"));
```

Comment:
I think we could change this code to the below so it can work in different platform.
```csharp
Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location),
    "TestData",
    "PhpWorkloads",
    "PutPhpWorkload.json");
```

Reply:
Thanks for the suggestion. Updated the code.